src/main/java/org/apache/sling/maven/feature/launcher/Launch.java

```diff
@@ -86,6 +89,14 @@ public void setEnvironmentVariables(Map<String, String> environmentVariables) {
         this.environmentVariables = environmentVariables;
     }
 
+    public List<String> getRepositoryUrls() {
```

Comment:
@rombert Can you adjust the javadoc in https://github.com/apache/sling-feature-launcher-maven-plugin/blob/869b9bbb3dc0c68527d5ca7294c616e8c5e4f206/src/main/java/org/apache/sling/maven/feature/launcher/StartMojo.java#L86 accordingly as well. Otherwise this is completely undocumented.